samples/bookInfoNoK8sPattern.yaml

```diff
@@ -52,7 +52,7 @@ services:
         labels:
           version: v1
       containers:
-      - image: docker.io/istio/examples-bookinfo-details-v1:1.16.2
```

Comment:
Best practice is to hardcode version which should work and not use latest. In that case it would be `1.16.4`

Reply:
Ok! Changing up!